src/Panel/Page.php

```diff
@@ -194,4 +186,83 @@ public function route(): array
             'title'      => $page->title()->toString(),
         ];
     }
+
+    /**
+     * Returns the next page in defined navigation
+     *
+     * @return \Kirby\Cms\Collection
+     */
+    public function next()
+    {
+        return $this->filter($this->model->nextAll($this->siblings()))->first();
+    }
+
+    /**
+     * Returns the prev page in defined navigation
+     *
+     * @return \Kirby\Cms\Collection
+     */
+    public function prev()
+    {
+        return $this->filter($this->model->prevAll($this->siblings()))->last();
+    }
+
+    /**
+     * Returns siblings of defined navigation
+     *
+     * @return \Kirby\Cms\Collection
+     */
+    public function siblings()
+    {
+        $page       = $this->model;
+        $model      = $page->parentModel();
+        $navigation = $page->blueprint()->navigation();
+        $sortBy     = $navigation['sortBy'] ?? null;
+        $status     = $navigation['status'] ?? null;
+
+        // if status is defined in navigation, all items in the collection are used (drafts, listed and unlisted)
+        // otherwise it depends on the status of the page
+        $collection = $status !== null ? $model->childrenAndDrafts() : $page->siblingsCollection();
```

Comment:
Wouldn't it be possible to simply use `$page->siblings()` here instead to avoid the change to siblingsCollection?

Reply:
Enlightening 😅